docs/python/editing.md

```diff
@@ -67,11 +72,48 @@ For help with common Intellisense and Python editing issues, check the table bel
 | Pylance is only offering top-level symbol options when adding imports. | By default, only top-level modules are indexed (depth =1). | Try increasing the depth to which Pylance can index your installed libraries through the `python.analysis.packageIndexDepths`. Check [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings). |
 | Pylance seems slow or is consuming too much memory when working on a large workspace. | Pylance analysis is done on all files present in a given workspace.  | If there are subfolders you know can be excluded from Pylance's analysis, you can add their paths to the `python.analysis.exclude` setting. Alternatively, you can try setting `python.analysis.indexing` to `false` to disable Pylance's indexer (**Note**: this will also impact the experience of completions and auto imports. Learn more about indexing in [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings)).  |
 | You are unable to install a custom module into your Python project. | The custom module is located in a non-standard location (not installed using pip). | Add the location to the `python.autoComplete.extraPaths` setting and restart VS Code. |
 
+#### Pylance Diagnostics
+Pylance by default provides diagnostics for Python files in the Problems panel.
+
+The list below are some of the most common diagnostics provided by Pylance and how to fix them.
+
+
+---
+##### `reportMissingModuleSource`
+This error occurs when Pylance is able to find type stubs for the imported package, but is unable find the package itself. This can happen when the package you are trying to import is not installed in the selected Python environment.
+
+**How to fix it**
+
+- If the package is already installed in a different interpreter or kernel, [select the correct interpreter](/docs/python/environments.md#select-and-activate-an-environment).
+- If the package is not installed, you can install it by running the following command in an activated terminal: `python -m pip install {package_name}`.
+
+---
+##### `reportMissingImports`
+
+This error happens when Pylance is unable to find the package or module you're importing, nor its type stubs.
+
+**How to fix it**
+- If you are importing a module, make sure it exists in your workspace or in a location that is included in the `python.autoComplete.extraPaths` setting.
+- If you are importing a package that is not installed, you can install it by running the following command in an activated terminal: `python -m pip install {package_name}`.
+- If you are importing a package that is already installed in a different interpreter or kernel, [select the correct interpreter](/docs/python/environments.md#select-and-activate-an-environment).
+
```

Comment:
Maybe we should also reference the 'editable' install situation here? There's a whole separate pyright documentation we could link to: https://microsoft.github.io/pyright/#/import-resolution?id=editable-installs